sdk/logic/Azure.ResourceManager.Logic/src/autorest.md

```diff
@@ -168,8 +168,6 @@ format-by-name-rules:
   'trackingId': 'uuid'
   'actionTrackingId': 'uuid'
   'PublicCertificate': 'any'
-  'content': 'any'
```

Comment:
`BinaryData` support string format, don't change this, if you got serialization issue please add double quotation marks

Reply:
these properties will throw JSON serialization exceptions because their content is an XML format string, like that [SWAGGER](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/examples/IntegrationAccountMaps_CreateOrUpdate.json#L11)

``` 
 System.Text.Json.JsonReaderException : '<' is an invalid start of a value. LineNumber: 0 | BytePositionInLine: 0.
```

Reply:
The SDK converts the read string in this logic below, but always throws an exception when converting an XML string
```
JsonSerializer.Serialize(writer, JsonDocument.Parse(Content.ToString()).RootElement);
``` 
#### example

```
var str = "value"
BinaryData Content = BinaryData.FromString($"\"{str}\"")
```
> Payload:  "content": "value"

```
var str = "<?xml version=\"2.0\" encoding=\"utf-8\" ?> \r\n"
BinaryData Content = BinaryData.FromString($"\"{str}\"")
```
> System.Text.Json.JsonReaderException : '2' is invalid after a single JSON value. Expected end of data. 

```
var str = "<?xml version='2.0' encoding='utf-8' ?>\r\n";
BinaryData Content = BinaryData.FromString($"\"{str}\"")
```
> System.Text.Json.JsonReaderException : '0x0D' is invalid within a JSON string. The string should be correctly escaped.

Reply:
Could you please try this sample and see if it can work?
```csharp
string val = "<?xml version=\"2.0\" encoding=\"utf-8\" ?> \r\n";
BinaryData data = BinaryData.FromObjectAsJson<string>(val);
```

Reply:
@Yao725 Thank you! That works